install/01-config.yaml

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
I wonder if the folder should be `sample-install` or something similar?

Reply:
done